test/fail_compilation/fix19059.d

```diff
@@ -2,10 +2,10 @@
  * PERMUTE_ARGS:
  * TEST_OUTPUT:
 ---
-fail_compilation/fix19059.d(16): Error: radix 8 digit expected, not `8`
-fail_compilation/fix19059.d(16): Error: octal literals `010` are no longer supported, use `std.conv.octal!10` instead
-fail_compilation/fix19059.d(17): Error: radix 8 digit expected, not `9`
-fail_compilation/fix19059.d(17): Error: octal literals `011` are no longer supported, use `std.conv.octal!11` instead
+fail_compilation/fix19059.d(16): Error: octal digit expected, not `8`
+fail_compilation/fix19059.d(16): Error: octal literals larger than 7 are no longer supported
+fail_compilation/fix19059.d(17): Error: octal digit expected, not `9`
+fail_compilation/fix19059.d(17): Error: octal literals larger than 7 are no longer supported
```

Comment:
Why did the `use std.conv.octal` disappear?

Reply:
None of the test cases show the `use` message. Can you please add one?

Reply:
I can add them. Note that the error messages exist in [other places](https://github.com/dlang/dmd/blob/79c8de1bdddfcd4c0338bde478d17eccb8edcd4b/test/fail_compilation/fail2656.d), this one test was added to check that `08` and `09` do not compile.